test/cql-pytest/test_alter_table.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests for alter table statement
+
+import time
+import pytest
+
+from util import new_test_table, unique_name
+from nodetool import flush
```

Comment:
I asked you earlier about flush, and you said you'd remove it and you did - but then you convinced me it **is** needed. So I guess you need to return it?
But if it's needed, how did the test pass without it?
(If it's not needed, you don't need to import it).

Reply:
If you look only at the test alone, the flush is not needed because we can assume (at least I thing so) the table will be only in memory.
But the test is run alongside other tests with test.py and we no longer can be sure that the table won't be dumped to the disk. So it'll be safer to add the flush.